ui-tests/src/main/java/org/example/pages/HomePage.java

```diff
@@ -0,0 +1,42 @@
+package org.example.pages;
+
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selectors.byId;
+import static com.codeborne.selenide.Selenide.*;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
do not use asssertThat in UI tests. there is no need to do it, use assertions from Selenide